### PR TITLE
Fix logic bug in Get-AzureRmVMDiskEncryptionStatus

### PR DESCRIPTION
**Pull request by ejarvi**
## Description

Fixes a logic bug in the case of VMs with only one encrypted volume type.  In particular, VMs with an OS volume and no data disks that had been encrypted with no AAD parameters.

## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [X] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [N/A] For public API changes to cmdlets:
    - [N/A] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [N/A] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by vladimir-shcherbakov** [comment]
on demand: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/927/
sign; https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/740/

**Comment by maddieclayton** [comment]
Doesn't need a sign job pass, merging.

File: src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/GetAzureDiskEncryptionStatus.cs [hunk 1]
```diff
@@ -514,7 +514,7 @@ public override void ExecuteCmdlet()
 
                 // retrieve encryption state using per-disk instance status and report if successful
                 AzureDiskEncryptionStatusContext encryptionStatus = GetStatusFromInstanceView();
-                if (encryptionStatus != null && !(encryptionStatus.OsVolumeEncrypted==EncryptionStatus.Unknown || encryptionStatus.DataVolumesEncrypted==EncryptionStatus.Unknown))
+                if (encryptionStatus != null && (encryptionStatus.OsVolumeEncrypted!=EncryptionStatus.Unknown || encryptionStatus.DataVolumesEncrypted!=EncryptionStatus.Unknown))
                 {
                     encryptionStatus.ProgressMessage = progressMessage;
                     WriteObject(encryptionStatus);
```
Comment (maddieclayton) [inline src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/GetAzureDiskEncryptionStatus.cs]: Please revert changes to the psm1 file below.

Comment (ejarvi) [reply src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/GetAzureDiskEncryptionStatus.cs]: reverted

**Comment by praries880** [comment]
on demand : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/930/

File: src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/GetAzureDiskEncryptionStatus.cs (hunk 1 above)
Comment (ejarvi) [reply src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/GetAzureDiskEncryptionStatus.cs]: @maddieclayton all checks have passed with the reverted file

**Pull request merged**